### PR TITLE
Add a shell commend to list loaded plugins.

### PR DESCRIPTION
**Pull request by dereuromark**
Right now you can only load plugins, but you are not sure if they are already loaded.
This command shows them pretty quick.

e.g.:

```
AssetCompress
Bake
CakePdf
DebugKit
Meta
Migrations
Setup
SocialShare
TinyAuth
Tools
```

### CODE REVIEW

**Comment by markstory** [comment]
Looks good but the conflicts need to be fixed.

**Comment by mamchenkov** [comment]
:+1:

File: src/Shell/PluginShell.php [hunk 1]
```diff
@@ -34,6 +35,15 @@ class PluginShell extends Shell
     ];
 
     /**
+     * @return void
```
Comment (ADmad) [inline src/Shell/PluginShell.php]: Description missing

**Comment by ADmad** [comment]
Tests needed.

**Comment by georgeconstantinou** [comment]
:+1:

**Comment by ionas** [comment]
Very handy and cool :+1: - Thanks.

**Comment by dereuromark** [comment]
Sometimes it's the small things :) You're welcome!

**Pull request merged**